vectorial_model.py

```diff
@@ -0,0 +1,141 @@
+import math
+from collections import Counter
+from collections import OrderedDict
+
+from collection_processing import article_tokenize_other, lemmatize, remove_stop_words
+
+
+def remove_non_index_term(query, inverted_index):
+    """"Remove tokens that are not in index from query."""
+    query_filtered = []
+    for token in query:
+        if token in inverted_index:
+            query_filtered.append(token)
+    return query_filtered
+
+
+def pre_processed_query(query, inverted_index, stop_words):
```

Comment:
There is already a function that is doing this: `process_text` in `collection_processing.py`

Reply:
This also removes the stop words from the query, but if we stop doing that the other function is equivalent.

Reply:
Sure. What I mean is that we should keep only one of the two functions and update it if needed. If we have twice the same code, it would diverge and create unwanted behaviors.